pkg/build/builder/util_linux.go

```diff
@@ -5,15 +5,22 @@ import (
 	"os"
 
 	"github.com/opencontainers/runc/libcontainer/cgroups"
-
 	s2iapi "github.com/openshift/source-to-image/pkg/api"
 )
 
 // GetCGroupLimits returns a struct populated with cgroup limit values gathered
 // from the local /sys/fs/cgroup filesystem.  Overflow values are set to
 // math.MaxInt64.
 func GetCGroupLimits() (*s2iapi.CGroupLimits, error) {
-	byteLimit, err := readInt64("/sys/fs/cgroup/memory/memory.limit_in_bytes")
+	// see https://git.kernel.org/pub/scm/linux/kernel/git/tj/cgroup.git/tree/Documentation/admin-guide/cgroup-v2.rst
+	// for list of cgroupv2 files to try, but Nalin relayed that examination of the crun and runc code that 'memory.high'
+	// is not used.
```

Comment:
Technically this comment belongs to cgroupv2 case inside the `if` block, but that's not important.

In the future, though, we might need to also check `memory.high` (in case `memory.max` returns `max`.